stack/stack_gfsv16.2.yaml

```diff
@@ -0,0 +1,359 @@
+cmake:
+  build: NO
+  bootstrap: NO
+  version: 3.20.1
+
+jpeg:
+  build: YES
+  shared: NO
+  version: 9.1.0
+
+zlib:
+  build: YES
+  shared: NO
+  version: 1.2.11
+
+libpng:
+  build: YES
+  shared: NO
+  version: 1.6.37
+
+szip:
+  build: YES
+  shared: NO
+  version: 2.1.1
+
+jasper:
+  build: YES
+  shared: NO
+  version: 2.0.25
+
+udunits:
+  build: YES
+  shared: NO
+  version: 2.2.28
+
+hdf5:
+  build: YES
+  version: 1.10.6
+  shared: NO
+  enable_szip: NO
+  enable_zlib: YES
+
+pnetcdf:
+  build: NO
+  version: 1.12.1
+  shared: NO
+
+netcdf:
+  build: YES
+  shared: NO
+  enable_pnetcdf: NO
+  version_c: 4.7.4
+  version_f: 4.5.3
+  version_cxx: 4.3.1
+
+nccmp:
+  build: YES
+  version: 1.8.9.0
+
+nco:
+  build: YES
+  version: 4.7.9
+
+cdo:
+  build: YES
+  version: 1.9.8
+
+pio:
+  build: YES
+  version: 2.5.3
+  enable_pnetcdf: NO
+  enable_gptl: NO
+
+esmf:
+  build: YES
+  version: 8_0_1
+  shared: NO
+  enable_pnetcdf: NO
+  debug: NO
+
+fms:
+  build: YES
+  version: 2021.03
+  repo: noaa-gfdl
+  cmake_opts: "-DGFS_PHYS=ON -D64BIT=ON -DOPENMP=ON"
+
+bacio:
+  build: YES
+  version: v2.4.1
+  install_as: 2.4.1
+  is_nceplib: YES
+
+sigio:
+  build: YES
+  version: v2.3.2
+  install_as: 2.3.2
+  is_nceplib: YES
+
+sfcio:
+  build: YES
+  version: v1.4.1
+  install_as: 1.4.1
+  is_nceplib: YES
+
+gfsio:
+  build: YES
+  version: v1.4.1
+  install_as: 1.4.1
+  is_nceplib: YES
+
+w3nco:
+  build: YES
+  version: v2.4.1
+  install_as: 2.4.1
+  is_nceplib: YES
+
+sp:
+  build: YES
+  version: v2.3.3
+  install_as: 2.3.3
+  openmp: ON
+  is_nceplib: YES
+
+ip:
+  build: YES
+  version: v3.3.3
+  install_as: 3.3.3
+  openmp: ON
+  is_nceplib: YES
+
+ip2:
+  build: YES
+  version: v1.1.2
+  install_as: 1.1.2
+  openmp: ON
+  is_nceplib: YES
+
+landsfcutil:
+  build: YES
+  version: v2.4.1
+  install_as: 2.4.1
+  is_nceplib: YES
+
+nemsio:
+  build: YES
+  version: v2.5.4
+  install_as: 2.5.4
+  is_nceplib: YES
+
+nemsiogfs:
+  build: YES
+  version: v2.5.3
+  install_as: 2.5.3
+  is_nceplib: YES
+
+w3emc:
+  build: YES
+  version: v2.9.2
+  install_as: 2.9.2
+  is_nceplib: YES
+
+g2:
+  build: YES
+  version: v3.4.5
+  install_as: 3.4.5
+  is_nceplib: YES
+
+g2c:
+  build: YES
+  version: v1.6.4
+  install_as: 1.6.4
+  is_nceplib: YES
+
+g2tmpl:
+  build: YES
+  version: v1.10.0
+  install_as: 1.10.0
+  is_nceplib: YES
+
+crtm:
+  build: YES
+  version: v2.4.0
+  install_as: 2.4.0
+  is_nceplib: YES
+  install_fix: YES
+
+upp:
+  build: YES
+  version: upp_v10.0.8
```

Comment:
@kgerheiser The upp tag would be upp_v8.1.1.  I have been waiting for updates of module variables from hpc-stack on Hera/Orion before tagging upp_v8.1.1.